lib/baseFactory.js

```diff
@@ -113,6 +112,31 @@ class BaseFactory {
     }
 
     /**
+     * Remove a Model
+     * @method remove
+     * @param  {Mixed}    config               Config object
+     * @return {Promise}
+     */
+    remove(config) {
```

Comment:
And why attach to a factory? Wouldn't it be easier just to attach the `remove()` function to each individual model?